response_operations_ui/views/messages.py

```diff
@@ -813,7 +814,10 @@ def _refine(message: dict, category: str = "SURVEY") -> dict:
         refined_message["from_internal"] = False
 
     if message.get("survey_id"):
-        refined_message["survey"] = get_survey_short_name_by_id(message.get("survey_id"))
+        if category != "SURVEY":
```

Comment:
Done a deeper dive and this isn't going to work. I also didn't fully appreciate the empty string is not really appropriate either, this is endpoint jumping due to poor endpoint creation, the empty string causes it to access a different endpoint, which we should avoid